hack/lib/golang.sh

```diff
@@ -270,7 +270,9 @@ kube::golang::create_gopath_tree() {
   rm -f "${go_pkg_dir}"
 
   # TODO: This symlink should be relative.
-  ln -s "${KUBE_ROOT}" "${go_pkg_dir}"
+  if [[ ! -e "${go_pkg_dir}" ]]; then
```

Comment:
Can you make this idempotent and non-destructive?  The unilateral `rm` above is bad in this case.

untested:

```
mkdir -p "${go_pkg_basedir}"
if [[ ! -e "${go_pkg_dir}" -o "$(readlink ${go_pkg_dir})" !=  "${KUBE_ROOT}" ]]; then 
    ln -sTf "${KUBE_ROOT}" "${go_pkg_dir}"
fi